packages/conventional-commit-lint/package.json

```diff
@@ -30,23 +31,28 @@
     "@commitlint/config-conventional": "^8.0.0",
     "@commitlint/lint": "^8.0.0",
     "@octokit/rest": "^16.28.3",
-    "gcf-utils": "^0.4.1",
-    "probot": "^9.2.15"
+    "gcf-utils": "^0.4.0",
+    "probot": "^9.2.20"
   },
   "devDependencies": {
+    "@types/body-parser": "^1.17.0",
```

Comment:
I think this should be in `dependencies` not `devDependencies`.

Reply:
I left this in the devDependencies as (in theory) one should only be using body-parser when running locally.

Reply:
oh, fair my bad; `@types` should usually be dev dependencies.